src/server/json_family.cc

```diff
@@ -164,8 +202,159 @@ OpResult<vector<OptSizeT>> OpStrLen(const OpArgs& op_args, string_view key, Json
   return vec;
 }
 
+OpResult<vector<OptSizeT>> OpObjLen(const OpArgs& op_args, string_view key, JsonExpression expression) {
+  OpResult<PrimeIterator> it_res = op_args.shard->db_slice().Find(op_args.db_ind, key, OBJ_STRING);
```

Comment:
it seems that lines 206-222 repeat themselves several times in this file. If I am not mistaken, then it's better to refactor them to a utility function that returns `OpResult<json>`